### ISSUE TITLE
weights update rule in cnn_train.m

### ISSUE THREAD

**Issue by xmaster302** [vlfeat/matconvnet#542]
I noticed that in the cnn_train.m  the weights is updated in this way:
```
% standard gradient training
      thisDecay = opts.weightDecay * net.layers{l}.weightDecay(j) ;
      thisLR = lr * net.layers{l}.learningRate(j) ;
      net.layers{l}.momentum{j} = ...
        opts.momentum * net.layers{l}.momentum{j} ...
        - thisDecay * net.layers{l}.weights{j} ...
        - (1 / batchSize) * res(l).dzdw{j} ;
      net.layers{l}.weights{j} = net.layers{l}.weights{j} + ...
        thisLR * net.layers{l}.momentum{j} ;
```
So,  net.layers{l}.weights{j} = net.layers{l}.weights{j} +thisLR * net.layers{l}.momentum{j} 
        = net.layers{l}.weights{j} + **thisLR * opts.momentum * net.layers{l}.momentum{j}**  - thisLR*(thisDecay * net.layers{l}.weights{j} - (1 / batchSize) * res(l).dzdw{j})

But according to the AlexNet paper,  **opts.momentum * net.layers{l}.momentum{j}** should not multiply by **thisLR** (the learning rate).  So, is  the weights update rule right in cnn_train.m?
